src/content/en/2020/caching.md

```diff
@@ -1,26 +1,26 @@
 part_number: IV
 chapter_number: 20
 title: Caching
 description: Caching chapter of the 2020 Web Almanac covering cache-control, expires, TTLs, validitaty, vary, set-cookies, AppCache, Service Workers and opportunities.
 authors: [roryhewitt, raghuramakrishnan71]
-reviewers: [csswizardry, jzyang, jaisanth, Soham-S-Sarkar]
+reviewers: [jzyang]
 analysts: [raghuramakrishnan71]
 translators: []
-roryhewitt_bio: Enterprise Architect at <a href="https://www.akamai.com/">Akamai</a>. Passionate about performance. Competition-level snowboarder. Multiple black belts. Motorcyclist. Boxer. Troublemaker. 
+roryhewitt_bio: Enterprise Architect at <a href="https://www.akamai.com/">Akamai</a>, who is passionate about performance. A British ex-patriate, he has lived in San Francisco for more than twenty years. In his spare time, he's a long-distance adventure motorcyclist, competition-level snowboarder and boxer with multiple black belts in various styles of karate. He likes being known as a troublemaker. Most importantly, he's a father and husband and the owner of Luna the cat. Find him at <a href="https://www.linkedin.com/in/roryhewitt/">LinkedIn</a>.
 raghuramakrishnan71_bio: Enterprise architect at <a href="https://www.tcs.com/">Tata Consultancy Services</a>, working on large digital transformation programs in the public sector. A technology enthusiast with a special interest in performance engineering. An avid traveler, intrigued by astronomy, history, biology, and advancements in medicine. A strong follower of the 47th verse, Chapter 2 of Bhagavad Gita "karmaṇy-evādhikāras te mā phaleṣhu kadāchana" meaning "You have a right to perform your prescribed duty, but you are not entitled to the fruits of action."
 
 discuss: 2056
 results: https://docs.google.com/spreadsheets/d/1fYmpSN3diOiFrscS75NsjfsrKXzxxhUMNcYSqXnQJQU/
 queries: 20_Caching
-#featured_quote: Caching benefits both the end users (they get their web pages quickly) and the companies serving the web pages (reducing the load on their servers). Caching really is a win-win!
-#featured_stat_1: TODO
-#featured_stat_label_1: TODO
-#featured_stat_2: TODO
-#featured_stat_label_2: TODO
-#featured_stat_3: TODO
-#featured_stat_label_3: TODO
+featured_quote: Caching provides a significant performance benefit by avoiding costly network requests - it helps both end users (they get their web pages quickly) and the companies serving web pages (reducing the load on their servers). Caching really is a win-win!
+featured_stat_1: 25.6%
+featured_stat_label_1: Percentage of HTTP responses don't include any caching information, which means they're probably being cached wrongly
+featured_stat_2: 21.4%
+featured_stat_label_2: Percentage of responses that aren't using revalidation, resulting in wasted bandwidth and increased latency
+featured_stat_3: 78.6%
+featured_stat_label_3: Percentage of sites could save up to 2MB of bandwidth by improving their caching policy
```

Comment:
This is what it looks like:

![image](https://user-images.githubusercontent.com/10931297/101818511-e0680000-3b1b-11eb-9856-e341b5f6510c.png)

I think a bit of a mouthful to be honest! Difficult to digest those stats quickly. Also some repetition in there (do we need to say it's a percentage when they have a percentage sign?)

Finally I think the 3rd stat is the wrong way of looking at this! Up to 2Mb of saving could mean 0.0001 Mb of saving. Let's flip it around and look at the ones over 2Mb instead of under (I also think we should change the chapter text to say this too).

```suggestion
featured_quote: Caching provides a significant performance benefit by avoiding costly network requests - it helps both end users (they get their web pages quickly) and the companies serving web pages (reducing the load on their servers). Caching really is a win-win!
featured_stat_1: 25.6%
featured_stat_label_1: HTTP responses with no caching information
featured_stat_2: 21.4%
featured_stat_label_2: Responses that cannot be revalidated
featured_stat_3: 21.3%
featured_stat_label_3: Sites that could save over 2MB on repeat visits with better caching
```

This looks like this, which I think is better:

![image](https://user-images.githubusercontent.com/10931297/101819654-76e8f100-3b1d-11eb-896d-67e63ef58c8f.png)

So I also think you should change this line in the text:

> Lighthouse also indicates how many bytes could be saved on repeat views by enabling a longer cache policy. Of the sites that could benefit from additional caching, 78.6% of them can reduce their page weight by up to 2MB!

to this:

> Lighthouse also indicates how many bytes could be saved on repeat views by enabling a longer cache policy. Of the sites that could benefit from additional caching, 21.3% of them can reduce their page weight by over 2MB!